test/test_brownian.py

```diff
@@ -146,33 +149,35 @@ def _eval(key):
     assert pval > 0.1
 
 
-@pytest.mark.parametrize("levy_area", ["", "space-time"])
-@pytest.mark.parametrize("use_levy", (False, True))
-def test_conditional_statistics(levy_area, use_levy):
+def conditional_statistics(
+    levy_area: _LA, use_levy: bool, tol=2**-6, spacing=2**-6, spline: _Spline = "sqrt"
```

Comment:
Let's remove the default arguments here as I don't think they're ever used.

This is a general principle -- prefer not to use default arguments where possible, in particular in internal APIs, as they're a common source of surprising behaviour.

Reply:
I know, thanks for the reminder. I guess old habits die hard.